### PR TITLE
#185106839 Setup Tailwind CSS processing on the client

### PR DESCRIPTION
**Pull request by divinecharlotte**
### What does this PR do?

- This PR is for Setting up Tailwind CSS processing on the client

### How should this be manually tested?

**Installation**
**1. Clone the repository by running** 
```
git clone https://github.com/atlp-rwanda/destructors-ec-fe.git
```
**2. Navigate to the project directory by running **
```
cd destructors-ec-fe
```
```
git checkout ch-setup-tailwindCss-185106839
```
**3. Install the necessary dependencies by running **
```
npm install
```
**6. Run react  development server by running**
```
npm run dev
```

### Any background context you want to provide?
N/A

### Questions:

- none

### CODE REVIEW

**Review by ba-lambert — APPROVED** [review]

**Review by katros1 — APPROVED** [review]

File: src/components/header.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```
Comment (leandreAlly) [inline src/components/header.jsx]: every components must start with capital letter as naming conventions.

File: src/App.jsx [hunk 2]
```diff
@@ -1,7 +1,5 @@
 import { useState } from 'react'
```
Comment (leandreAlly) [inline src/App.jsx:1]: can you create your components for testing styling purpose not adding your styles into the App.jsx

**Review by leandreAlly — CHANGES_REQUESTED** [review]

**Review by buka1calvin — APPROVED** [review]

**Review by leandreAlly — APPROVED** [review]

**Review by JENNY-257 — APPROVED** [review]

**Review by mjeanbosco19 — APPROVED** [review]

**Review by Abdulkeza — APPROVED** [review]
Nice work

**Review by Abdulkeza — COMMENTED** [review]
Nice work, but you need to solve conflicts

**Pull request merged**